src/visualization/types/Mosaic/options.tsx

```diff
@@ -37,18 +43,12 @@ const MosaicOptions: FC<Props> = ({properties, results, update}) => {
     return results.table.getColumnType(k) === 'string'
   })
 
-  const numericColumns = results.table.columnKeys.filter(key => {
-    if (key === 'result' || key === 'table') {
-      return false
-    }
-
-    const columnType = results.table.getColumnType(key)
-
-    return columnType === 'time' || columnType === 'number'
-  })
+  // Mosaic graphs are currently limited to always using _time on the x-axis
+  //   future enhancements will depend on the visualization library
+  const xDataColumn = results.table.columnKeys.filter(key => key === '_time')
```

Comment:
is this deep chain of components for sure to exist or do we need to perform some null checks?

Reply:
It does exist. But doesn't hurt to add the optional chaining syntax. The codebase is not consistent in this regard. Some graph types use the optional chaining syntax while others do not.

I will use it here and we can address the other graph types separately.